lib/react/jsx/processor.rb

```diff
@@ -0,0 +1,17 @@
+module React
+  module JSX
+    class Processor
+      def self.instance
+        @instance ||= new
+      end
+      
+      def self.call(input)
+        instance.call(input)
+      end
+
+      def call(input)
+        {data: JSX::transform(input[:data])}
```

Comment:
from https://github.com/rails/sprockets#return-hash:

> Returning a String is shorthand for returning `{ data: str }`

Can we return a string only? Do we need any metadata?